geomap/src/types/index.ts

```diff
@@ -28,7 +28,7 @@ export interface MapNode {
 export interface Alarm {
   id: string
   severity: string
-  nodeId: number
+  nodeId: string
```

Comment:
Is there a reason you switched this to a string? It should be a number...

Reply:
Yeah, both number and string will work. I just tried to make it consistent with the node id in the "Node" in the same file that is migrated from foundation-ui.

Reply:
My suspicion is that `node` is a field that can be either a `nodeId` (number) _or_ a `foreignSource:foreignId` pair. But in this case, `nodeId` should always be the node ID, which is a number.

Reply:
It's probably because everything UI receives is a string, so I guess it's not that matter if it's consistent with the backend. I just updated it to be number just now. Please check.